lib/systems/doubles.nix

```diff
@@ -29,7 +29,7 @@ let
     "armv7l-linux" "i686-linux" "m68k-linux" "microblaze-linux"
     "microblazeel-linux" "mipsel-linux" "mips64el-linux" "powerpc64-linux"
     "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "s390-linux"
-    "s390x-linux" "x86_64-linux"
+    "s390x-linux" "x86_64-linux" "loongarch64-linux"
```

Comment:
Please keep these alphabetically sorted.

Reply:
Got it.